numba/core/types/misc.py

```diff
@@ -144,14 +144,15 @@ class CPointer(Type):
     """
     mutable = True
 
-    def __init__(self, dtype):
+    def __init__(self, dtype, addrspace=None):
```

Comment:
What is the expected type for `addrspace`?

Reply:
The expected type of `addrspace` is int.

Reply:
I added the docstring according to how I know best. I did not find any docstring detailing the other argument `dtype` in the same class, so my addition might not be correct. Please advise if the format or the convention is incorrect.

Reply:
Format-wise, we haven't defined a styleguide but I prefer the numpydoc style like: https://github.com/numba/numba/blob/5f7bcce048b9c0a7fd02376ca8a2e3574625c99f/numba/core/errors.py#L665-L668
For reference: https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
Changed to fit numpydoc style.